test/validate_events.test.js

```diff
@@ -59,7 +59,7 @@ describe('Events as given in /_data/events', () => {
 describe('Invalid events given in /test/invalid_events', () => {
   describe('that require at least 1 character: ', () => {
```

Comment:
the description does not match the test case. I'd propose you delete the second level `describe`. So the reason the tests are invalid is reflected in the name of the json file.